addons/point_of_sale/static/src/app/store/pos_store.js

```diff
@@ -1721,10 +1721,9 @@ export class PosStore extends Reactive {
         }
     }
 
-    async getRenderedReceipt(order, title, lines, fullReceipt = false, diningModeUpdate) {
+    getPrintingChanges(order, diningModeUpdate) {
```

Comment:
You cannot really changes args in stable. And normally now, we cannot add "feature" in stable